stack.yaml

```diff
@@ -1,26 +1,19 @@
-resolver: lts-13.26
+resolver: lts-17.6
 # Resolves hjsmin requiring language-javascript ==0.6, while the purescript
 # compiler requires language-javascript ==0.7.
 allow-newer: true 
 packages:
 - '.'
 extra-deps:
+- barrier-0.1.1
 - bytestring-trie-0.2.5.0
-- cborg-0.2.2.0
-- connection-0.3.1
-- happy-1.19.9
+- classy-prelude-yesod-1.5.0
```

Comment:
Why is barrier and class-prelude-yesod being added?

Reply:
I'd guess because they're not in lts-17.6 at the versions we want